br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -439,28 +257,13 @@ func (sr *SchemasReplace) rewriteTableInfo(value []byte, dbID int64) ([]byte, er
 		for i, tbl := range partitions.Definitions {
 			newID, exist := tableReplace.PartitionMap[tbl.ID]
 			if !exist {
-				newID, exist = sr.globalTableIdMap[tbl.ID]
-				if !exist {
-					if sr.IsRestoreKVStatus() {
-						return nil, errors.Annotatef(berrors.ErrInvalidArgument, "failed to find id:%v in maps", tbl.ID)
-					}
-
-					newID, err = sr.genGenGlobalID(context.Background())
-					if err != nil {
-						return nil, errors.Trace(err)
-					}
-					sr.globalTableIdMap[tbl.ID] = newID
-				}
-				tableReplace.PartitionMap[tbl.ID] = newID
+				// partition filtered out
+				return nil, nil
```

Comment:
continue or return here?

Reply:
thanks for the review. good call! should continue, but since we don't have partition filter, it should not be triggered at all. Let me add a log at here